tests-integration/src/cluster.rs

```diff
@@ -59,6 +59,7 @@ pub struct GreptimeDbClusterBuilder {
     cluster_name: String,
     kv_backend: KvBackendRef,
     store_config: Option<ObjectStoreConfig>,
+    custom_store_types: Option<Vec<StorageType>>,
```

Comment:
If we choose `providers` in `StoreConfig`, then we can rename this into `store_providers` too.